tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -850,6 +851,39 @@ func printActionDeleted(entityName string) {
             }))
 }
 
+// Check if the specified action is a web-action
+func IsWebAction(client *whisk.Client, qname *QualifiedName) bool {
```

Comment:
Don't need to pass the QualifiedName as a pointer as we run the risk of modifying the original content by accident. Also camel casing for `isWebAction`.